src/trajectory.jl

```diff
@@ -322,8 +308,8 @@ function randcat(rng, zs::AbstractVector{<:PhasePoint}, unnorm_ℓP::AbstractMat
     return z
 end
 
-function sample_phasepoint(rng, τ::StaticTrajectory{MultinomialTS}, h, z)
-    n_steps = abs(τ.termination_criterion.L)
+function sample_phasepoint(rng, τ::Trajectory{MultinomialTS}, h, z)
+    n_steps = abs(nsteps(τ))
```

Comment:
```suggestion
    n_steps = nsteps(τ)
```

Reply:
This will not support negative step sizes any more.

Reply:
Ah of course, sorry!